__tests__/ssr/app/entry-server.ts

```diff
@@ -1,8 +1,16 @@
+import Vue from 'vue'
 import { createApp } from './main'
+import { PiniaSsr, getRootState } from '../../../src'
+
+// simulate server-side environment
+// @ts-ignore
+delete global.window
```

Comment:
we should be able to add 

```js
/**
 * @jest-environment node
 */
```

to the top of the `spec.ts` file to deactivate jsdom

Reply:
thanks :) didn't know that